lte/gateway/python/integ_tests/s1aptests/BUILD.bazel

```diff
@@ -0,0 +1,489 @@
+    tags = TAG_EXTENDED_TEST,
+    deps = [":s1ap_wrapper"],
+)
+
+pytest_test(
+    name = "test_paging_after_mme_restart",
+    size = "small",
+    srcs = ["test_paging_after_mme_restart.py"],
+    imports = [LTE_ROOT],
+    tags = TAG_EXTENDED_TEST + TAG_TRAFFIC_SERVER_TEST,
+    deps = [":s1ap_wrapper"],
+)
+
+pytest_test(
+    name = "test_attach_nw_initiated_detach_fail",
+    size = "small",
+    srcs = ["test_attach_nw_initiated_detach_fail.py"],
+    imports = [LTE_ROOT],
+    tags = TAG_EXTENDED_TEST,
+    deps = [":s1ap_wrapper"],
+)
+
+pytest_test(
+    name = "test_tau_ta_updating",
+    size = "small",
+    srcs = ["test_tau_ta_updating.py"],
+    imports = [LTE_ROOT],
+    tags = TAG_EXTENDED_TEST,
+    deps = [":s1ap_wrapper"],
+)
+
+pytest_test(
+    name = "test_tau_ta_updating_reject",
+    size = "small",
+    srcs = ["test_tau_ta_updating_reject.py"],
+    imports = [LTE_ROOT],
+    tags = TAG_EXTENDED_TEST,
+    deps = [":s1ap_wrapper"],
+)
+
+pytest_test(
+    name = "test_tau_mixed_partial_lists",
+    size = "small",
+    srcs = ["test_tau_mixed_partial_lists.py"],
+    imports = [LTE_ROOT],
+    tags = TAG_EXTENDED_TEST,
+    deps = [":s1ap_wrapper"],
+)
+
+pytest_test(
+    name = "test_eps_bearer_context_status_multiple_ded_bearer_deact",
+    size = "small",
+    srcs = ["test_eps_bearer_context_status_multiple_ded_bearer_deact.py"],
+    imports = [LTE_ROOT],
+    tags = TAG_EXTENDED_TEST,
+    deps = [":s1ap_wrapper"],
+)
+
+pytest_test(
+    name = "test_guti_attach_with_zero_mtmsi",
+    size = "small",
+    srcs = ["test_guti_attach_with_zero_mtmsi.py"],
+    imports = [LTE_ROOT],
+    tags = TAG_EXTENDED_TEST,
+    deps = [":s1ap_wrapper"],
+)
+
+pytest_test(
+    name = "test_ics_timer_expiry_with_mme_restart",
+    size = "small",
+    srcs = ["test_ics_timer_expiry_with_mme_restart.py"],
+    imports = [LTE_ROOT],
+    tags = TAG_EXTENDED_TEST,
+    deps = [":s1ap_wrapper"],
+)
+
+pytest_test(
+    name = "test_attach_mobile_reachability_timer_expiry",
+    size = "large",
+    srcs = ["test_attach_mobile_reachability_timer_expiry.py"],
+    imports = [LTE_ROOT],
+    tags = TAG_EXTENDED_TEST,
+    deps = [":s1ap_wrapper"],
+)
+
+pytest_test(
+    name = "test_attach_implicit_detach_timer_expiry",
+    size = "large",
+    srcs = ["test_attach_implicit_detach_timer_expiry.py"],
+    imports = [LTE_ROOT],
+    tags = TAG_EXTENDED_TEST,
+    deps = [":s1ap_wrapper"],
+)
+
+pytest_test(
+    name = "test_mobile_reachability_tmr_with_mme_restart",
+    size = "large",
+    srcs = ["test_mobile_reachability_tmr_with_mme_restart.py"],
+    imports = [LTE_ROOT],
+    tags = TAG_EXTENDED_TEST,
+    deps = [":s1ap_wrapper"],
+)
+
+pytest_test(
+    name = "test_implicit_detach_timer_with_mme_restart",
+    size = "large",
+    srcs = ["test_implicit_detach_timer_with_mme_restart.py"],
+    imports = [LTE_ROOT],
+    tags = TAG_EXTENDED_TEST,
+    deps = [":s1ap_wrapper"],
+)
+
+pytest_test(
```

Comment:
Do we really want the setup and tear-down to be modeled as a `pytest_test` target? If we execute all tests with `bazel ...` it will not execute due to the manual tag, but any query for e.g. `bazel query "kind(pytest_.*, //orc8r/... union //lte/...)"` does return these as well.

Reply:
What would you suggest as an alternative?
In the query you can exclude manual tests.
The setup and teardown are still tests if you look at the code. IMO if you want to change it to something else you should change the code too to be a normal script.

Reply:
Since we have the specific tags 
```
    "TAG_EXTENDED_TEST_SETUP",
    "TAG_EXTENDED_TEST_TEARDOWN",
```
I guess we are distinguishing them already. Agree, perhaps we should create a task to refactor these?